test/e2e/pod_create_test.go

```diff
@@ -1068,4 +1068,47 @@ ENTRYPOINT ["sleep","99999"]
 
 	})
 
+	It("podman pod create --share-parent test", func() {
+		podCreate := podmanTest.Podman([]string{"pod", "create", "--share-parent=false"})
+		podCreate.WaitWithDefaultTimeout()
+		Expect(podCreate).Should(Exit(0))
+
+		ctrCreate := podmanTest.Podman([]string{"create", "--pod", podCreate.OutputToString(), ALPINE})
+		ctrCreate.WaitWithDefaultTimeout()
+		Expect(ctrCreate).Should(Exit(0))
+
+		podStart := podmanTest.Podman([]string{"pod", "start", podCreate.OutputToString()})
+		podStart.WaitWithDefaultTimeout()
+		Expect(podStart).Should(Exit(0))
+
+		inspectPod := podmanTest.Podman([]string{"pod", "inspect", podCreate.OutputToString()})
+		inspectPod.WaitWithDefaultTimeout()
+		Expect(inspectPod).Should(Exit(0))
+		data := inspectPod.InspectPodToJSON()
+
+		inspect := podmanTest.InspectContainer(ctrCreate.OutputToString())
+		Expect(data.CgroupPath).To(HaveLen(0))
+		Expect(inspect[0].HostConfig.CgroupParent).To(Equal(data.CgroupPath))
```

Comment:
why do we expect that container cgroup parent is equal to the pod cgroup path?  Shouldn't it happen only with `--share-parent=true`?

Reply:
@giuseppe `share-parent` is true by default as was the old default case.

Reply:
`podCreate := podmanTest.Podman([]string{"pod", "create", "--share-parent=false"})`

aren't we setting it to false here though?